src/examples/browser.yml

```diff
@@ -1,6 +1,6 @@
 description: >
-  Runs Cypress tests in specific browser.
-  Installs dependencies and caches NPM modules and the Cypress binary.
+  Run Cypress tests using specified browser. The (circleci/browser-tools)[https://circleci.com/developer/orbs/orb/circleci/browser-tools#commands-install-browser-tools]
```

Comment:
I don't think yaml understands this markdown link syntax. It just shows as text.